tests/foreman/api/test_syncplan.py

```diff
@@ -776,11 +777,9 @@ def test_positive_synchronize_custom_product_future_sync_date(self):
         self.validate_repo_content(
             repo, ['erratum', 'package', 'package_group'], after_sync=False)
         # Create and Associate sync plan with product
-        if bz_bug_is_open(1695733):
-            self.logger.info('Need to set seconds to zero because BZ#1695733')
-            sync_date = datetime.utcnow().replace(second=0) + timedelta(seconds=delay)
-        else:
-            sync_date = datetime.utcnow() + timedelta(seconds=delay),
+        self.logger.info('Need to set seconds to zero because BZ:1695733')
+        # sync_date = datetime.utcnow() + timedelta(seconds=delay)
```

Comment:
Why is old code left commented? Shouldn't we use these `BEGIN: END` markers here?